nnpackage/examples/one_op_in_tflite/metadata/MANIFEST

```diff
@@ -1,7 +1,8 @@
 {
   "major-version" : "1",
-  "minor-version" : "0",
+  "minor-version" : "1",
   "patch-version" : "0",
+  "config"        : "config.cfg",
```

Comment:
@seanshpark What about using a list like `"configs" : [ "config.cfg" ]` for multiple configuration? We may use 1st item by default or explicitly provides the index for others.

Reply:
OK, nice to have multiples. Then we need an index to provide... do you any idea how to provide the index ?
maybe another item like `"cfxidx" : "2",` ?

Reply:
Or is it like one configuration file for one model ?

Reply:
@seanshpark the index of "configs" list could be used like we did in "models". We need to choose whether the configuration will be applied to all models in an nnpackage or per-model. If latter, we need to provide 2-dimension array since we allow multiple models in an nnpackage.

Reply:
> OK, nice to have multiples. Then we need an index to provide... do you any idea how to provide the index ?
> maybe another item like `"cfxidx" : "2",` ?

Maybe I've misunderstood. If you're talking how the user can select the index, we need to extend or provide a new API.

Reply:
`config` here is to override environment variables, like `RUY_THREADS=4` or `RUY_THREADS=1`
while looking at `ConfigSource.cc`, when one configure is loaded to provide `getConfigOrDefault()` values, 
I don't know how we can change it before running a model... so per-model is a bit hard to implement righnow for me.

Reply:
@seanshpark I think your suggestion is better since it does not require API work. :)

```
"models" : [ "m1.tflite", "m2.circle", "m3.circle", "m4.tflite" ]
"configs" : [ ["m1cfg"], [ ] , ["m3debug", "m3mixed" ] ]
"config-idx" : [ 0, -1, 1, -1 ] 
```
- `config-idx` is the zero-based index from `configs`.
- `-1` in `config-idx` means _"use default configuration"_.

Reply:
From off-line discussion, as for now, we'll go with array of `configs` and array of `config-idx` with one element for now. extending this will be left for the future.

Reply:
For `config-idx` it should be always `[ 0 ]` for now, I'll leave this implementation for the future when this is needed.